websites/S/Stats.fm/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "973027832307535952"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+function getElementByXpath(path: string) {
+	return document.evaluate(
+		path,
+		document,
+		null,
+		XPathResult.FIRST_ORDERED_NODE_TYPE,
+		null
+	).singleNodeValue;
+}
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "stats_fm",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location;
+	switch (true) {
+		case pathname === "/": {
+			presenceData.details = "Browsing the main page";
+			break;
+		}
+		case pathname.includes("/album/"): {
+			presenceData.details = "Viewing an album";
+			presenceData.state = `${
+				getElementByXpath("/html/body/div[1]/div[1]/div/div/div[2]/h1")
+					.textContent
+			}`;
```

Comment:
```suggestion
			presenceData.state = document.querySelector("h1.text-center").textContent;
```

Reply:
I will do it at 3:00 pm (Canadian time) because im having school right now if it would be okey?